test/node-unit/cli/run-helpers.spec.js

```diff
@@ -1,8 +1,71 @@
 'use strict';
 
-const {validatePlugin, list} = require('../../../lib/cli/run-helpers');
+const {
+  validatePlugin,
+  list,
+  loadRootHooks
+} = require('../../../lib/cli/run-helpers');
+
+describe('helpers', function() {
+  describe('loadRootHooks()', function() {
+    describe('when passed nothing', function() {
+      it('should reject', async function() {
+        return expect(loadRootHooks(), 'to be rejected');
+      });
+    });
+
+    describe('when passed empty array of hooks', function() {
+      it('should return an empty MochaRootHooks object', async function() {
+        return expect(loadRootHooks([]), 'to be fulfilled with', {
+          beforeAll: [],
+          beforeEach: [],
+          afterAll: [],
+          afterEach: []
+        });
+      });
+    });
+
+    describe('when passed an array containing hook objects and sync functions and async functions', function() {
+      it('should flatten them into a single object', async function() {
+        function a() {}
+        function b() {}
+        function d() {}
+        function g() {}
+        function f() {}
+        function c() {
+          return {
+            beforeAll: d,
+            beforeEach: g
+          };
+        }
+        async function e() {
+          return {
+            afterEach: f
```

Comment:
Do you have a test that f` is async?

Reply:
heh, I'll just add the `async` keyword to that one I guess.